.github/workflows/integration-tests.yml

```diff
@@ -102,24 +102,60 @@ jobs:
 
   psycopg2-check:
     name: "Test psycopg2 build version"
-    runs-on: ${{ matrix.scenario.platform }}
+    runs-on: ${{ matrix.platform }}
     strategy:
       fail-fast: false
       matrix:
-        scenario:
-          - {platform: ubuntu-latest, psycopg2-name: psycopg2}
-          - {platform: macos-12, psycopg2-name: psycopg2-binary}
+        platform:
+          - ubuntu-24.04
```

Comment:
Is there any reasonable way for us to test that:
```
pip install dbt-postgres
pip uninstall psycopg2-binary
pip install psycopg2
```
Works on Linux (assuming appropriate build dependencies are available)?